testing/travis.py

```diff
@@ -125,15 +124,25 @@ def run_cmds(cmds, fail_sequential=False):
     return to_return
 
 
-def filter_not_docfiles(diff_names):
-
-    is_doc = lambda f: f.startswith('docs') or any([f.endswith(ext) for ext in ['md', 'rst']])
-    return [f for f in diff_names.split() if not is_doc(f)]
+def skip(diff_names):
+    if os.environ['TRAVIS_SECURE_ENV_VARS'] == 'false':
```

Comment:
I always prefer Python's dictionary `.get` method, since it never causes a traceback. Using it here has the added advantage of making this script mildly less travis-specific.